gprofiler/profilers/python.py

```diff
@@ -163,6 +152,19 @@ def make_application_metadata(self, process: Process) -> Dict[str, Any]:
         return metadata
 
 
+@register_profiler(
+    "Python",
+    profiler_name="PySpy",
+    # py-spy is like pyspy, it's confusing and I mix between them
+    possible_modes=["auto", "pyspy", "py-spy"],
```

Comment:
What does "auto" mean? This profiler no longer supports any "modes", it's just py-spy.